docs/tables/kubernetes_{custom_resource_singular_name}.md

```diff
@@ -157,7 +157,7 @@ Steampipe will automatically create the `kubernetes_certificate` table, which ca
 +--------------------+--------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 ```
 
-```bash
```

Comment:
This `bash` works because the content does not have single/double quotes anywhere. So you could leave it in (although no difference between leaving it in and removing it).

Reply:
Thanks, @chandru89new, that makes sense.